### PR TITLE
[SystemZ] Add missing (dis-)assembly tests.

### PR DESCRIPTION
**Pull request by dominik-steenken**
### Assembly Tests
This adds assembly tests for 
```
lcdfr, lndfr, lpdfr
```

### Disassembly Tests
This also adds a number of disassembly tests for the following menmonics:
```
cgit, cgrt, cit, clfit, clgit, clgrt, clgt, clrt, clt, crt, lcdfr, lndfr, lpdfr
```
For `cutfu` / `cu12`, it also adds a check for bad register choice.

### CODE REVIEW

**Comment by dominik-steenken** [comment]
@uweigand FYI

File: llvm/test/MC/Disassembler/SystemZ/insns.txt [hunk 1]
```diff
@@ -10656,18 +10752,6 @@
 # CHECK: lxdbr %f13, %f15
 0xb3 0x05 0x00 0xdf
 
-# CHECK: lxdr %f0, %f8
```
Comment (uweigand) [inline llvm/test/MC/Disassembler/SystemZ/insns.txt:10659]: Why are you removing these?  That isn't mentioned in the commit message at all ...

File: llvm/test/MC/Disassembler/SystemZ/insns-z14.txt [hunk 2]
```diff
@@ -989,6 +995,9 @@
 # CHECK: vflcsb %v14, %v17
 0xe7 0xe1 0x00 0x00 0x24 0xcc
 
+# CHECK: vflls %v0, %v1
```
[the remark this answers is not in the document]
Comment (dominik-steenken) [reply llvm/test/MC/Disassembler/SystemZ/insns-z14.txt:998]: Actually, this is covered by `vldeb` and should be deleted.

File: llvm/test/MC/Disassembler/SystemZ/insns.txt (hunk 1 above)
Comment (dominik-steenken) [reply llvm/test/MC/Disassembler/SystemZ/insns.txt]: my apologies, it seems i messed up the commits i aggregated into this PR. I believe i have now fixed the issues mentioned.

**Review by uweigand — APPROVED** [review]
LGTM now, thanks!

**Pull request merged**